inc/toolbox.class.php

```diff
@@ -2088,10 +2088,11 @@ static function parseMailServerConnectString($value, $forceport = false) {
     *
     * @param $value String host connect string ex
     *                      {localhost:993/imap/ssl}INBOX
+    * @param $new boolean (@since 9.3.1) is the parent form is for a new item
```

Comment:
The `@since` tag should be move outside the `@param`tag (see http://docs.phpdoc.org/references/phpdoc/tags/since.html).